index.html

```diff
@@ -41,6 +42,7 @@
 <script src="libs/mode-solidity.js"></script>
 <script src="bin/soljson-latest.js"></script>
 <script src="libs/ethereumjs-vm.js"></script>
+<script src="libs/universal-dapp.js"></script>
 <script src="libs/web3.min.js"></script>
```

Comment:
@chriseth Regarding upgrading web3.js to latest release, you mentioned before that you;d made changes to the the one in lib is there a commit to reference for the changes required? 27938b4 shows using it but not the modifications made